src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -479,24 +479,61 @@ function prepareConstSumQuestions() {
                 numResponses = parseInt(numResponses, 10);
 
                 $(`#constSumInfo-${qnNum}-${numResponses - 1}`).show();
+            } else {
+                $(`[id^="constSumInfo-${qnNum}-"]`).show();
+                $(`[id^="constSumSubmissionFormOptionFragment-${qnNum}"]`).addClass('padding-left-55px');
+                $(`[id^="constSumSubmissionFormOptionFragment-${qnNum}"]`).addClass('margin-top-15px');
             }
         } else {
             $(`[id^="constSumInfo-${qnNum}-"]`).hide();
         }
     }
 }
+function updateConstSumInstructionsQn(qnNum) {
+    let numOptions = 0;
+    let points = parseInt($(`#constSumPoints-${qnNum}`).val(), 10);
+    const numRecipients = parseInt($(`[name="questionresponsetotal-${qnNum}"]`).val(), 10);
+    const distributeToRecipients = $(`#constSumToRecipients-${qnNum}`).val() === 'true';
+    const pointsPerOption = $(`#constSumPointsPerOption-${qnNum}`).val() === 'true';
+
+    if (distributeToRecipients) {
+        numOptions = numRecipients;
+    } else {
+        numOptions = parseInt($(`#constSumNumOption-${qnNum}`).val(), 10);
+    }
+
+    if (pointsPerOption) {
+        points *= numOptions;
+    }
+
+    const constSumDistributePointsOptionSelected = $(`#constSumDistributePointsOptions-${qnNum}`).val();
+    const forceAllUnevenDistribution = constSumDistributePointsOptionSelected
+            === ParamsNames.FEEDBACK_QUESTION_CONSTSUMALLUNEVENDISTRIBUTION;
+    const forceSomeUnevenDistribution = constSumDistributePointsOptionSelected
+            === ParamsNames.FEEDBACK_QUESTION_CONSTSUMSOMEUNEVENDISTRIBUTION;
+    const instructionElement = $(`#constSumInstruction-${qnNum}`);
+    let instruction = '';
+    const infoIcon = '<span class="glyphicon glyphicon-info-sign padding-right-10px"></span>';
+
+    instruction += `${infoIcon} Total points distributed should add up to ${points}.<br>`;
+    if (forceAllUnevenDistribution) {
+        instruction += `${infoIcon} Every ${distributeToRecipients ? 'recipient' : 'option'} `
+                + 'should be allocated different number of points.<br>';
+    } else if (forceSomeUnevenDistribution) {
+        instruction += `${infoIcon} At least one ${distributeToRecipients ? 'recipient' : 'option'} `
+                + 'should be allocated different number of points.<br>';
+    }
+    instructionElement.html(instruction);
+}
 
 function updateConstSumMessageQn(qnNum) {
     let numOptions = 0;
     let points = parseInt($(`#constSumPoints-${qnNum}`).val(), 10);
     const numRecipients = parseInt($(`[name="questionresponsetotal-${qnNum}"]`).val(), 10);
     const distributeToRecipients = $(`#constSumToRecipients-${qnNum}`).val() === 'true';
     const pointsPerOption = $(`#constSumPointsPerOption-${qnNum}`).val() === 'true';
-    const forceUnevenDistribution = $(`#constSumUnevenDistribution-${qnNum}`).val() === 'true';
```

Comment:
Ok i'll look into this

Reply:
So I went through the PR which was helpful. Based on my understanding, `forceUnevenDistribution` was the variable used to mention `All options are different` before `Some options` feature came along. I think the reason the reminders aren't working in the old version is because I have completely eliminated that variable in my version. So I'll include it as well in the if conditions to ensure it is backward compatible. Hopefully that should resolve it